pandas/io/excel/_base.py

```diff
@@ -681,13 +681,14 @@ class ExcelWriter(metaclass=abc.ABCMeta):
         be parsed by ``fsspec``, e.g., starting "s3://", "gcs://".
 
         .. versionadded:: 1.2.0
-    if_sheet_exists : {'error', 'new', 'replace'}, default 'error'
+    if_sheet_exists : {'error', 'new', 'replace', 'write_to'}, default 'error'
```

Comment:
why does this api exist here at all? why is this not on .to_excel?

Reply:
It's _also_ there... Ill have a look and maybe remove it...

Reply:
Well, the PR (and all previous PRs) are about adding the option to the *ExcelWriter*, not to the .to_excel.... Of course I could change that, or maybe add it even, but that would be another PR

Reply:
However, now it is an option with the ExcelWriter, the user cannot set it differently for every separate DataFrame.... That is a choice for discussion (except it is already in the version)

Reply:
ok, fine for this PR, @rhshadrach can you have a look here. we ought to simply deprecate ExcelWriter

Reply:
I do not find "write_to" to be clear - write to what? I prefer "overwrite" instead.

Reply:
write to the existing sheet. because it is an option for "if_sheet_exists".
I personally found the difference between "overwrite" and "replace" a bit ambiguous: Do we overwrite the sheet with the dataframe? No, we write the DataFrame to the already existing sheet. For example, if I "overwrite" a text document/CSV, I do not write the data to it, but rather overwrite the entire sheet.
Maybe a "write_cells" or "overwrite_cells" would be more clear?

Reply:
I agree on the confusion between overwrite and place, but don't think write_cells nor overwrite_cells addresses it. What "add_to"?

Reply:
I actually quite like overlay. Very intuitive